prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-information-page/ldap-information-page.component.html

```diff
@@ -65,8 +65,8 @@
           </app-alert>
 
           <ng-template #contact>
-            please contact Reg Admin at <a href="tel:12509529137">(250) 952-9137</a> or
-            <a href="mailto:HLTH.REGISTRIESADMIN@gov.bc.ca">HLTH.REGISTRIESADMIN@gov.bc.ca</a>.
+            please contact the HelpDesk at <a href="tel:12509521234">(250) 952-1234</a> or
```

Comment:
Is `HelpDesk` the name or is it supposed to have a space between the words?

Reply:
Possibly the name, its cp what was in the ticket